### ISSUE TITLE
Progress bar not working for a range scan

### ISSUE THREAD

**Issue by matteobsu** [McStasMcXtrace/McCode#439]
On my workstation (Windows 10 x64), the progress bar is not working correctly when simulating for a range of values of one paremeters. In fact it is not updating and it's displayed only when the current paramater value has been finished simulating. 

To reproduce the "bug" you can try simulate any instrument with one parameter e.g par= 0,1, in 10 steps.

**Comment by matteobsu** [comment]
As an example this is the print after it finish simulating for SDD = 0.9
![capture](https://cloud.githubusercontent.com/assets/15745811/24900607/b06dd43e-1ea4-11e7-8623-f5b738d81659.PNG)
however while it's still simulating, for example for the new value of SDD = 1.149, the progress bar is not displayed.
![capture1](https://cloud.githubusercontent.com/assets/15745811/24900670/d7854174-1ea4-11e7-98d0-4e64e9a09e2a.PNG)

**Comment by willend** [comment]
Dear @matteobsu,

I belive that you have stumbled upon an 'intended feature' rather than a bug: 

The idea was simply to not overload the terminal with output in case of doing a scan.

As you probably know we are replacing the Perl tools with a similar set of Python tools. The python mcrun has a "verbose" option that would show the information you are after (plus other "noise" ;-) ).

<pre>
mcrun --help
loading user configuration from /Users/pkwi/.mcstas/2.4pre02/mccode_config.json
Usage: mcrun.py [-cpnN] Instr [-sndftgahi] params={val|min,max|min,guess,max}...

Options:
  --version             show program's version number and exit
  -h, --help            show this help message and exit

  mcrun options:
    -c, --force-compile
                        force rebuilding of instrument
    -p FILE, --param=FILE
                        read parameters from file FILE
    -N NP, --numpoints=NP
                        set number of scan points
    -L, --list          use a fixed list of points for linear scanning
    -M, --multi         run a multi-dimensional scan
    --autoplot          open plotter on generated dataset
    --mpi=NB_CPU        spread simulation over NB_CPU machines using MPI
    --machines=machines
                        defines path of MPI machinefile to use in parallel
                        mode
    --optimise-file=FILE
                        store scan results in FILE (defaults to: "mccode.dat")
    --no-cflags         disable optimising compiler flags for faster
                        compilation
    --verbose           enable verbose output

  Instrument options:
    -s SEED, --seed=SEED
                        set random seed (must be: SEED != 0)
    -n COUNT, --ncount=COUNT
                        set number of neutrons to simulate
    -t, --trace         enable trace of neutron through instrument
    -g, --gravitation   enable gravitation for all trajectories
    -d DIR, --dir=DIR   put all data files in directory DIR
    --format=FORMAT     output data files using format FORMAT (format list
                        obtained from <instr>.out -h)
    --no-output-files   Do not write any data files
    -i, --info          Detailed instrument information
</pre>

- Hence I will close this issue with status 'wontfix'.
